### PR TITLE
track the status of decoratd jobs

### PR DESCRIPTION
**Pull request by aojea**
The job enables the `checkconfig` options to warn on decorated jobs

```
       - --warnings=non-decorated-jobs
        - --warnings valid-decoration-config
 ```

 is kept optional and not reporting, but will give us a place to track when new jobs using bootstraputils are added

### CODE REVIEW

File: config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml [hunk 1]
```diff
@@ -72,6 +72,33 @@ presubmits:
...
+        - --warnings=tide-strict-branch
+        - --warnings=needs-ok-to-test
+        - --warnings=validate-owners
+        - --warnings=missing-trigger
+        - --warnings=validate-urls
+        - --warnings=unknown-fields
+        - --warnings=duplicate-job-refs
+        - --warnings=unknown-fields-all
+        - --warnings=non-decorated-jobs
+        - --warnings valid-decoration-config
```
Comment (ameukam) [inline config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml:99]:
```suggestion
        - --warnings=valid-decoration-config
```

**Review by BenTheElder — COMMENTED** [review]
eventually we should fold these warnings into the main job, but it's probably too noisy right now

File: config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml (hunk 1 above)
Comment (BenTheElder) [reply config/jobs/kubernetes/test-infra/test-infra-presubmits.yaml]: yeah, the space won't work here because this is a single argv entry

**Pull request merged**